docs/application/dotnet/guides/app-management/component-manager.md

```diff
@@ -0,0 +1,142 @@
+
+# Component Manager
+
+Component manager provides information about installed and running components.
+
+The main features of the `Tizen.Applications.ComponentBased.ComponentManager` class include:
+
+-   Managing running component context
```

Comment:
Similar suggestion with  https://github.com/Samsung/tizen-docs/pull/840#discussion_r330386233.

@surbhig12 would you please check the comment and share your opinion with us?

Reply:
Done